components/HowItWorks/HowItWorks.tsx

```diff
@@ -63,15 +72,30 @@ export function HowItWorks() {
           }
           content={
             <>
-              You are staking{" "}
+              {isDelegate ? (
+                <Tooltip
+                  label={
+                    delegatorAddress
+                      ? `Delegator address ${delegatorAddress}`
+                      : ""
+                  }
+                >
+                  <>
```

Comment:
wrap this `<Link>` in a `<span>` instead of a fragment to fix the issue. the tooltip wants a real a dom node, breaks with the virtual one from the fragment

Reply:
awesome will try

Reply:
that fixed it!